### PR TITLE
Update instructions on updating vimrc

### PR DESCRIPTION
**Pull request by fazlearefin**

### CODE REVIEW

**Comment by elliotwutingfeng** [comment]
Maybe it would be better to use `git stash` instead of `git reset` as according to [this page](https://www.theserverside.com/definition/git-stash), a reset creates a new commit point in the branch history (probably undesirable?); stash does not.

**Comment by fazlearefin** [comment]
> Maybe it would be better to use `git stash` instead of `git reset` as according to [this page](https://www.theserverside.com/definition/git-stash), a reset creates a new commit point in the branch history (probably undesirable?); stash does not.

`git reset --hard` does not create any new commit

File: README.md [hunk 1]
```diff
@@ -63,8 +63,9 @@ If you have vim aliased as `vi` instead of `vim`, make sure to either alias it:
 Just do a git rebase!
 
     cd ~/.vim_runtime
+    git reset --hard
```
Comment (bostonaholic) [inline README.md:66]:
```suggestion
    git reset --hard
    git clean -d --force
```

`git reset` doesn't remove untracked files. Use `git clean` for this.

**Comment by amix** [comment]
Thanks, folks 🙏

**Pull request merged**